karma.coverage.js

```diff
@@ -4,7 +4,12 @@ const karmaConfig = require('./karma.base.js');
 
 karmaConfig.webpack = require('./webpack.config.coverage.js');
 
-karmaConfig.browsers = ['PhantomJS'];
+if (process.env.TRAVIS) {
+  karmaConfig.browsers = ['PhantomJS'];
```

Comment:
Can we use the Chrome browser instead like we've started doing on other properties? (https://github.com/theonion/avclub/blob/master/karma.conf.js#L80)